.circleci/config.yml

```diff
@@ -111,13 +111,51 @@ jobs:
           working_directory: ./frontend/
       - run:
           name: Test Code
-          command: npm test -- --reporters=jest-junit
+          command: |
+            mkdir --parents /tmp/workspace/test-results/frontend-coverage
+            npm test -- \
+              --ci \
+              --coverage \
+              --coverageDirectory=/tmp/workspace/test-results/frontend-coverage \
+              --reporters=jest-junit \
+              --reporters=default
```

Comment:
I'd set the `--reports` and `--coverage` options directly in `jest.config.js`, since it's fine having those enabled locally as well (at least at this point in time, when the overhead is still pretty low), and having CI be less of a snowflake will make the effects of future modifications to the test script more predictable.